mosc.js

```diff
@@ -47,7 +47,13 @@ var MoscBase = function (evaluation_context_dictionary)
 		    {
 		    	try
 		    	{
-		    		propertyBase[property_parts[0]] = eval(get_eval_string(pvalue));
+				// arbitary code execution mitigation
+				let blacklist = ['&', ';', '|', '-', '$', '`', '||'];
+				if (blacklist.some(v => get_eval_string(pvalue).includes(v))) {
+					break;
```

Comment:
I'm not a fan of using `break;` to mitigate security issues.
Security fix should be insensitive and shouldn't make additional execution branches.

Reply:
Thank you @toufik-airane for the suggestion.
I have updated the fix, and now instead of terminating when a blacklisted character is detected, it sanitizes the `eval()` string by removing the blacklisted character. So it now shows an error instead of terminating(which is intended). I've run the mocha tests and the fix doesn't break anything else.

Thanks,
Asjid